js/components/utils.js

```diff
@@ -496,9 +496,13 @@ var Utils = {
         currentProp = props[i];
         currentObject = value = currentObject[currentProp];
       }
-      return value || defaltValue;
+
+      if (value === undefined) {
```

Comment:
Value could be null (for example, if currentObject[currentProp] is null), and we would return null instead of the defaultValue. Is this what we wanted?

Reply:
I think that since `null` is deliberately set, it could be considered to be the property value and thus shouldn't be replaced with the default, although in practice it might cause some confusion. Either way works for me as long as we specify it in the jsdoc, but I agree that we should use `typeof value === 'undefined'`

Reply:
I'm going to use hasOwnProperty instead, since you could set it to anything and what we really care about is that the key exists in the object.

Reply:
I'm going with your solution because the hasOwnProperty has issues with variables from prototypes.